src/test/serialization_tests.cpp

```diff
@@ -39,66 +39,58 @@ using tensorflow::TensorProto;
 using tensorflow::serving::PredictRequest;
 using tensorflow::serving::PredictResponse;
 
-using InferenceEngine::IInferRequest;
-using InferenceEngine::PreProcessInfo;
-using InferenceEngine::ResponseDesc;
-
 using namespace ovms;
 using namespace InferenceEngine;
 
 using testing::_;
 using testing::NiceMock;
 using testing::Throw;
 
-const std::vector<InferenceEngine::Precision> SUPPORTED_OUTPUT_PRECISIONS{
-    // InferenceEngine::InferenceEngine::Precision::UNSPECIFIED,
-    // InferenceEngine::Precision::MIXED,
-    InferenceEngine::Precision::FP32,
-    InferenceEngine::Precision::FP16,
-    // InferenceEngine::Precision::Q78,
-    InferenceEngine::Precision::I16,
-    InferenceEngine::Precision::U8,
-    InferenceEngine::Precision::I8,
-    InferenceEngine::Precision::U16,
-    InferenceEngine::Precision::I32,
-    InferenceEngine::Precision::I64,
-    // InferenceEngine::Precision::BIN,
-    // InferenceEngine::Precision::BOOL
-    // //InferenceEngine::Precision::CUSTOM)
+const std::vector<ovms::Precision> SUPPORTED_OUTPUT_PRECISIONS_2{
+    // ovms::Precision::UNDEFINED,
+    // ovms::Precision::MIXED,
+    ovms::Precision::FP32,
+    ovms::Precision::FP16,
+    // ovms::Precision::Q78,
+    ovms::Precision::I16,
+    ovms::Precision::U8,
+    ovms::Precision::I8,
+    ovms::Precision::U16,
+    ovms::Precision::I32,
+    ovms::Precision::I64,
+    // ovms::Precision::BIN,
+    // ovms::Precision::BOOL
+    // ovms::Precision::CUSTOM)
 };
 
-const std::vector<InferenceEngine::Precision> UNSUPPORTED_OUTPUT_PRECISIONS{
-    InferenceEngine::Precision::UNSPECIFIED,
-    InferenceEngine::Precision::MIXED,
-    // InferenceEngine::Precision::FP32,
-    // InferenceEngine::Precision::FP16,
-    InferenceEngine::Precision::Q78,
-    // InferenceEngine::Precision::I16,
-    // InferenceEngine::Precision::U8,
-    // InferenceEngine::Precision::I8,
-    // InferenceEngine::Precision::U16,
-    // InferenceEngine::Precision::I32,
-    // InferenceEngine::Precision::I64,
-    InferenceEngine::Precision::BIN,
-    InferenceEngine::Precision::BOOL
-    // InferenceEngine::Precision::CUSTOM),
+const std::vector<ovms::Precision> UNSUPPORTED_OUTPUT_PRECISIONS_2{
+    // ovms::Precision::UNDEFINED, // Cannot create blob with such precision
+    // ovms::Precision::MIXED, // Cannot create blob with such precision
+    // ovms::Precision::FP32,
+    // ovms::Precision::FP16,
+    // ovms::Precision::Q78, // Cannot create blob with such precision
+    // ovms::Precision::I16,
+    // ovms::Precision::U8,
+    // ovms::Precision::I8,
+    // ovms::Precision::U16,
+    // ovms::Precision::I32,
+    // ovms::Precision::I64,
+    // ovms::Precision::BIN, // Cannot create blob with such precision
+    ovms::Precision::BOOL
+    // ovms::Precision::CUSTOM),
```

Comment:
Leave TODO here - there's more supported precisions in OV 2.0 API now. We should make sure all are enlisted here as either supported or unsupported.